util/openocd/target/lowrisc-earlgrey-lc.cfg

```diff
@@ -0,0 +1,29 @@
+# Copyright lowRISC contributors.
```

Comment:
This separate config is definitely the way to go!

I noticed that the IDCODE differs from [e2e_bootstrap_rma](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/tests/rom/e2e_bootstrap_rma/src/main.rs;l=240;drc=4995811929c102aedc1fc9b34e3b1b5f4887c367). Maybe I'm missing context, but if they're both trying to connect to the LC TAP, shouldn't they have the same values?

I don't actually speak Verilog, but this is why I believed it would be hardcoded to 1: https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/lc_ctrl/rtl/lc_ctrl.sv;l=210;drc=5d1d047b96abb7f5022b5e12a5596ce012f5e6e7

Reply:
Thanks, that makes sense! Filed https://github.com/lowRISC/opentitan/issues/17634 to track fixing the incorrect TAP ID.

Reply:
@dmcardle: once this openocd/jtag infra is in opentitanlib, we can refactor your bootstrap RMA test to use it rather than embed all the TCL script in the host-side test program itself.